### DISCUSSION TITLE
<title>BUG : `Starstruck` achievement animation is buggy

### DISCUSSION THREAD

**Discussion by RushilJalal** [community/community#129569] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have noticed 'Starstruck' achievement animation is buggy. 

https://github.com/community/community/assets/54212357/c7d9ac45-af82-4a1c-b033-5e55cd78a4be

**Comment by ebndev** [comment]
Hi @RushilJalal, thanks for being a part of the GitHub Community!

We appreciate you taking the time to share this.

As a valued GitHub user, your feedback in this Discussion space, along with appropriate tags and examples, plays a crucial role in shaping the development of GitHub. While we can’t guarantee immediate action on every piece of feedback, please know that we are attentive to your suggestions and continuously strive to enhance the GitHub experience.
